api/types/accountrecovery.go

```diff
@@ -1,4 +1,4 @@
-/**
+/*
```

Comment:
For reference, addlicense uses `// ` in all lines. A bunch of these "older" headers predates addlicense, or have been manually copied around.

Reply:
Thanks - I'll see if I can update the tool or it's configuration.

Reply:
I don't recommend fighting addlicense, I would just let it do it's thing. We have a few mixed comment styles, but in the end it isn't a big deal.